actions.livemd

```diff
@@ -19,97 +19,64 @@ Mix.install([{:ash, "~> 3.0.0-rc"}], consolidate_protocols: false)
 <a class="flex ml-2" style="color: #61758a;" href="overview.livemd">Home</a>
 </div>
 <div class="flex">
-<a class="flex mr-2" style="color: #61758a;" href="domain.livemd">Domain</a>
+<a class="flex mr-2" style="color: #61758a;" href="querying.livemd">Querying</a>
 <i class="ri-arrow-right-fill"></i>
 </div>
 </div>
 
 <!-- livebook:{"break_markdown":true} -->
 
-### In this tutorial, you will create a Resource with a 'create' action
+Actions and domains are much simpler: an action describes a <span title="Create, Read, Update, Delete">CRUD</span> operation that can be performed on a resource, but using whatever language makes the most sense for your given resource.  For example, you may have a `publish` action on a `BlogPost` resource that changes the resource to be publicly visible.  Under CRUD, this would be an `update` operation, but calling it the "publish action" more fully conveys what the operation does.
```

Comment:
```suggestion
Actions are much simpler: an action describes a <span title="Create, Read, Update, Delete">CRUD</span> operation that can be performed on a resource, but using whatever language makes the most sense for your given resource.  For example, you may have a `publish` action on a `BlogPost` resource that changes the resource to be publicly visible.  Under CRUD, this would be an `update` operation, but calling it the "publish action" more fully conveys what the operation does.
```

Seems like the `and domains` bit isn't necessary?

Reply:
Oops!  Yes, this was left over from an earlier draft.